litellm/tests/test_completion.py

```diff
@@ -2330,6 +2330,37 @@ def test_completion_gemini():
 
 # test_completion_gemini()
 
+# test Gemini with base64 images
+def test_completion_gemini_vision_base64():
+    try:
+        litellm.set_verbose = True
+        image_path = "../proxy/cached_logo.jpg"
+        # Getting the base64 string
+        base64_image = encode_image(image_path)
+        resp = litellm.completion(
+            model="gemini/gemini-pro-vision",
```

Comment:
shouldn't this be for `gemini/gemini-pro` since the if-block would allow any gemini-pro model to make this call?